lib/TriggerResult.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Represents the result of a call to {@link Pusher::trigger}.
+ */
+class TriggerResult {
```

Comment:
Please keep ``Pusher`` in class names to avoid conflicts with other libraries (PHP class names must be unique).

The common convention for classes which are not using namespaces is to use underscores as separators (similar to the way namespaces would be used for 5.3+): ``Pusher_TriggerResult``